api/v1/aerospikecluster_validating_webhook.go

```diff
@@ -1813,10 +1816,15 @@ func validateRequiredFileStorageForFeatureConf(
 	// CA cert related fields are not supported with Secret Manager, so check their mount volume
 	allPaths = append(allPaths, caPaths...)
 
+	if defaultPassFilePath != nil {
```

Comment:
We can add a condition here to error out in case of secret agent default password file path